ui/partials/perf/runinfo.html

```diff
@@ -0,0 +1,3 @@
+<span ng-if="runs.length > 0" class="run-detail" tooltip="Runs: < {{runs.join(', ')}} >">
```

Comment:
That was what I thought, though @avih felt that square brackets made 1's hard to read:

https://github.com/mozilla/treeherder-ui-deprecated/pull/532#issuecomment-103576766

It looks like commas were left out deliberately as well. Oops. I'll add them back. I'm not sure if there's any science on this so I think I'll just keep things the way they were before.